[Fundamentals] Bulk enable features (#14457)
cloudflare/cloudflare-docs: content/fundamentals/basic-tasks/basic-features.md

---
pcx_content_type: reference

> title: Enable features in bulk

Reviewer: ```suggestion
title: Enable basic features in bulk
```

---

> # Enable features in bulk

Reviewer: ```suggestion
# Enable basic features in bulk
```

> Use the **Basic Features** toggle to activate 10 core speed and security features included in your plan.

Reviewer: ```suggestion
Use the **Basic Features** toggle to activate 10 core speed and security features included in your domain plan.
```

## Included features

> The following features can be enabled from **Basic Features**:

Reviewer: ```suggestion
The following features can be activated from **Basic Features**:
```

- [HTTP/2 to Origin](/speed/optimization/protocol/http2-to-origin/)
- [HTTP/3 (with QUIC)](/speed/optimization/protocol/http3/)
- [0-RTT Connection Resumption](/speed/optimization/protocol/0-rtt-connection-resumption/)

> - [Always Use HTTPs](/ssl/edge-certificates/additional-options/always-use-https/)

Reviewer 2: ```suggestion
- [Always Use HTTPS](/ssl/edge-certificates/additional-options/always-use-https/)
```

- [TLS 1.3](/ssl/edge-certificates/additional-options/tls-13/)
- [Normalize incoming URLs](/rules/normalization/)
- [WebSockets](/network/websockets/)
- [Onion Routing](/network/onion-routing/)
- [Browser Integrity Check](/waf/tools/browser-integrity-check/)
- [Hotlink Protection](/waf/tools/scrape-shield/hotlink-protection/)

## Activate basic features

> To enable several key features in bulk:

Reviewer: ```suggestion
To enable several basic features in bulk:
```

1. Log in to the [Cloudflare dashboard](https://dash.cloudflare.com).

> 2. Select your account and zone.

Reviewer: ```suggestion
2. Select your account and domain.
```

> 3. On the zone homepage, go to **Quick Actions**.

Reviewer: ```suggestion
3. On the selected domain's **Overview** page, go to **Quick Actions**.
```

4. For **Basic Features**, select **Activate**.

> 5. Choose which features you want to be enabled.

Reviewer: ```suggestion
5. Choose which features you want to be activated.
```
